plugin/pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -34,6 +34,11 @@ type SchedulerExtender interface {
 	// are used to compute the weighted score for an extender. The weighted scores are added to
 	// the scores computed  by Kubernetes scheduler. The total scores are used to do the host selection.
 	Prioritize(pod *v1.Pod, nodes []*v1.Node) (hostPriorities *schedulerapi.HostPriorityList, weight int, err error)
+
+	// Bind informs the extender about the scheduling decision, before the decision is
+	// conveyed to the apiserver. The binding information contains the pod and the
+	// selected node.
+	Bind(pod *v1.Pod, node string) error
```

Comment:
Yes, this behavior is no different with k8s apiserver. If the node became unschedulable after the scheduler selected it, the binding can fail and the pod is scheduled again.